flipperdevices/flipperzero-firmware
# Gui: scrollable long file names in FileBrowser and Archive Browser

# What's new

- Gui: scrollable long file names in FileBrowser
- Archive:  scrollable long file names

# Verification 

- Compile and upload
- Check any application that uses File Browser
- Check long names in Archive

# Checklist (For Reviewer)
